README.md

```diff
@@ -1,90 +1,80 @@
-# Atom Community
- 
-> To get involved with Atom Community please reach out on the [Discord Server](https://discord.gg/2tD9evh8qP) or within the [Discussions Tab](https://github.com/atom-community/atom/discussions). The issues tab is only used for the technical issues.
-
-
-> Due to changes in the upstream, the original Atom and its repositories will be archived on December 15, 2022. If you'd like to learn more about the official Atom archiving, you can read about it in their [official announcement](https://github.blog/2022-06-08-sunsetting-atom/).
-
-[![Build Status](https://dev.azure.com/atomcommunity/atomcommunity/_apis/build/status/atom-community/Release%20Branch%20Build?branchName=master)](https://dev.azure.com/atomcommunity/atomcommunity/_build/latest?definitionId=10&branchName=master)
 
-Atom is a hackable text editor for the 21st century, built on [Electron](https://github.com/electron/electron), and based on everything we love about our favorite editors. We designed it to be deeply customizable, but still approachable using the default configuration.
+<div align = center>
 
-![Atom](https://user-images.githubusercontent.com/378023/49132477-f4b77680-f31f-11e8-8357-ac6491761c6c.png)
+[![Banner]][#]
```

Comment:
Last thing!

The `#` href [links to the top of the page](https://weblog.west-wind.com/posts/2019/Jan/21/NonNavigating-Links-for-JavaScript-Handling#dont-use--by-itself) which seems more unexpected than useful practically

So:
```suggestion
![Banner]
```

Or if the image shouldn't have a link:
```suggestion
[![Banner]][#dontlink]
```

<details><summary>Btw, (doesn't matter)</summary>

it's easier to type
```md
something\
newline
```

than

```md
something<br>
newline
```

(The backslash "escapes" the newline)

Might be helpful in the future
</details>

Reply:
> #

Is what people usually use because:
- On some pages it actually makes the image unclickable
- On the pages it doesn't, it will just stay on the same page

> something\ 

Less reliable since you then have to be sure
to not have any spaces afterwards, the same
way you do with C/++ macros.